src/Microsoft.Identity.Client/AppConfig/PublicClientApplicationBuilder.cs

```diff
@@ -68,6 +68,23 @@ public static PublicClientApplicationBuilder Create(string clientId)
             return new PublicClientApplicationBuilder(config).WithClientId(clientId);
         }
 
+        /// <summary>
+        /// Default redirect URI for desktop interactive public client applications. 
+        /// </summary>
+        /// <example>
+        /// If your application uses AcquireTokenInteractive, you want to register the
+        /// <c>https://login.microsoftonline.com/common/oauth2/nativeclient</c> redirect URI
+        /// and build your application using
+        /// <c>.WithRedirectUri(PublicClientApplicationBuilder.DefaultInteractiveDesktopRedirectUri)</c>
+        /// <code>
+        /// IPublicClientApplication app;
+        /// app = PublicClientApplicationBuilder.Create(clientId)
+        ///         .WithRedirectUri(PublicClientApplicationBuilder.DefaultInteractiveDesktopRedirectUri)
+        ///         .Build();
+        /// </code>
+        /// </example>
+        public const string DefaultInteractiveDesktopRedirectUri = "https://login.microsoftonline.com/common/oauth2/nativeclient";
```

Comment:
Is this the best place to put the const?

Reply:
Probably not. Any idea of an ideal place?